Bio/Entrez/__init__.py

```diff
@@ -132,6 +139,30 @@
 api_key = None
 
 
+def format_ids(ids):
+    """Convert one or more UIDs to a single comma-delimited string.
+
+    Input may be a single ID as an integer or string, an iterable of strings/ints,
+    or a string of IDs already separated by commas.
+    """
+    if isinstance(ids, int):
+        # Single integer, just convert to str
+        return str(ids)
+
+    if isinstance(ids, str):
+        # String which represents one or more IDs joined by commas
+        # Remove any whitespace around commas if they are present
+        return ",".join(id.strip() for id in ids.split(","))
+
+    # Not a string or integer, assume iterable
+    return ",".join(map(str, ids))
+
+
+def count_ids(ids_str):
```

Comment:
This function is not covered by the tests, and does not seem worth adding to the public API in my opinion.

Reply:
Should have been private to begin with, but since its only used in one spot I just removed the function altogether.